kube-controllers/tests/fv/fv_test.go

```diff
@@ -114,7 +114,7 @@ var _ = Describe("kube-controllers FV tests", func() {
 		Eventually(func() *api.ClusterInformation {
 			info, _ = calicoClient.ClusterInformation().Get(context.Background(), "default", options.GetOptions{})
 			return info
-		}).ShouldNot(BeNil())
+		}, 10*time.Second).ShouldNot(BeNil())
```

Comment:
We were only giving the controller 1 second to launch and create this resource, which was racy and caused flaky tests.